docker/README.md

```diff
@@ -62,12 +62,90 @@ docker-compose -f docker-compose-mysql.yml down
 
 Please note that SQL support is still in active developement and it is not production ready yet.
 
-Quickstart for production
+One host production setup with docker-compose
 =========================
-In a typical production setting, dependencies (cassandra / statsd server) are
-managed / started independently of the cadence-server. To use the container in
-a production setting, use the following command:
+For a simple production setup, start with our **docker-compose-production-example.yml** for a single host Cadence cluster as described in the following steps.
```

Comment:
There are a couple things missing here. The first is that there is no explanation of why the yaml file is set to 0.5.9 version of Cadence. If that's done on purpose to have them walk through the upgrade scenario, then call that out.

The other thing missing is some indication of when the schema upgrade is needed. Is there anything they should look for in the release notes? Should they consider a change in minor version to indicate a schema change?

Reply:
Also, we should have a MySQL version of the docker-compose. Most people will be using MySQL.

Reply:
I would update that 0.5.9 to 0.x.y-auto-setup and let them choose the latest one to start with.
 
Currently there is no place to know whether or not a schema changes is needed. I will include that into our release process. Let us make sure very release telling it. If they run with old schema, the server won't start successfully. 

Agreed that we should have a MySQL version. Issue created https://github.com/uber/cadence/issues/2406 We need to practice how to use the commands to do backup like Cassandra